src/main/java/frc/robot/RobotContainer.java

```diff
@@ -84,6 +91,6 @@ private MoInput getInput() {
     }
 
     public Command getAutonomousCommand() {
-        return Commands.print("No autonomous command configured");
+        return PathPlannerCommands.getFollowPathCommand(drive, positioning, autoPathName, true);
```

Comment:
You're hard-coding `true` for `shouldAssumeRobotIsAtStart`. While this made sense last year, we might want to reconsider now that we have access to dynamic repathing. Maybe we shouldn't _always_ assume the robot is at the start, and instead let it use apriltag positioning to figure out its actual position. Then it could repath and account for any offset between the ideal starting position and it's actual position on the field.

What makes the most sense to me is to add a switch to the shuffleboard. Then we can evaluate on a case-by-case position which option we'd like to use.